v3-mongodb-v3-sql/transform.js

```diff
@@ -13,19 +13,11 @@ function transformEntry(entry, model) {
   Object.keys(entry).forEach((key) => {
     const attribute = model.attributes[key];
 
-    if (toOmit.includes(key)) {
-      return;
-    }
-
-    if (!attribute) {
-      return;
-    }
-
     if (
       key === "createdAt" &&
       (model.options.timestamps === true ||
         _.isUndefined(model.options.timestamps) ||
-        model.options.timestamps[0] === "created_at")
+        model.options.timestamps.includes('createdAt'))
```

Comment:
actually this was intended as some users might already have customized the fields to be created_at so your fix will break that usecase. I will probably rewrite part of it anyway and push a bit later if you don't mind